src/main/java/com/rackspace/salus/resource_management/services/ResourceManagement.java

```diff
@@ -160,8 +189,11 @@ public boolean exists(String tenantId, String resourceId) {
    * @param presenceMonitoringEnabled Whether presence monitoring is enabled or not.
    * @return Stream of resources.
    */
-  public Stream<Resource> getResources(boolean presenceMonitoringEnabled) {
-    return resourceRepository.findAllByPresenceMonitoringEnabled(presenceMonitoringEnabled).stream();
+  public Stream<ResourceDTO> getResources(boolean presenceMonitoringEnabled) {
+    return resourceRepository.findAllByPresenceMonitoringEnabled(presenceMonitoringEnabled)
+        .stream()
+        .map(r -> getResourceDTOFromResource(r))
```

Comment:
IntelliJ might be suggesting to you
```suggestion
        .map(this::getResourceDTOFromResource)
```